<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPort</class>
 <widget class="QMainWindow" name="SerialPort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerialPort</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_receive">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始接收指令</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>91</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="b.qrc">
      <normaloff>:/new/prefix1/b.png</normaloff>:/new/prefix1/b.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>210</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_left">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="icon">
     <iconset resource="l.qrc">
      <normaloff>:/new/prefix1/l.png</normaloff>:/new/prefix1/l.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>210</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_forward">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>91</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="f.qrc">
      <normaloff>:/new/prefix1/f.png</normaloff>:/new/prefix1/f.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>210</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_right">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="r.qrc">
      <normaloff>:/new/prefix1/r.png</normaloff>:/new/prefix1/r.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>210</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="s.qrc">
      <normaloff>:/new/prefix1/s.png</normaloff>:/new/prefix1/s.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>331</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string> 指令</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="b.qrc"/>
  <include location="f.qrc"/>
  <include location="l.qrc"/>
  <include location="r.qrc"/>
  <include location="s.qrc"/>
 </resources>
 <connections/>
</ui>
